api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceProducer.java

```diff
@@ -28,11 +28,14 @@
 import javax.enterprise.inject.Produces;
 import javax.inject.Inject;
 
+import com.datastax.driver.core.Session;
```

Comment:
Minor: could you fix your IDE settings to follow the import ordering rules? There are some shared settings which you can import:
https://github.com/hawkular/hawkular-build-tools/tree/master/ide-configs